packages/cloud_firestore/cloud_firestore/windows/cloud_firestore_plugin.cpp

```diff
@@ -38,6 +39,7 @@ using flutter::EncodableValue;
 
 namespace cloud_firestore_windows {
 
+static std::string kLibrarayName = "flutter-fire-fst";
```

Comment:
```suggestion
static std::string kLibraryName = "flutter-fire-fst";
```
Just a spelling mistake I've noticed for this static that needs to also be changed for auth and storage. I also think perhaps calling it `flutter-fire-firestore` would be better, assuming this would appear as part of the logs. `fst` wouldn't be immediatelt clear what library it is referring to IMO.